### PR TITLE
Fix docbuild warnings

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by swewers** [comment]
I have simultaneously tried to fix the errors in the docs. I found some more errors (which do not give a warning) and fixed them too.

Should I push these changes into this pull request (if so, how?) or should I just wait until you merge this and then do my own pull request?

File: docs/semistable_reduction.rst [hunk 1]
```diff
@@ -6,3 +6,5 @@ Semistable reduction of curves
     admissible_reduction
     superp_models
     reduction_trees
+    superell
+    superp
```
Comment (saraedum) [inline docs/semistable_reduction.rst]: @swewers I added these back in so the documentation builds. Did you mean to drop these modules from the documentation? If so, then we also need to delete the corresponding `.rst` files.

Comment (swewers) [reply docs/semistable_reduction.rst]: the modules `superell` and `superp` are now obsolete, so the corresponding doc files should be deleted.

**Comment by saraedum** [comment]
You could add your changes by pushing to the branch on my repository or by creating a pull request into my branch. Both are somewhat tricky so it's probably easier to merge this first and then create a new PR with your changes.

**Pull request merged**